tutorials/io/saving_games.rst

```diff
@@ -118,6 +118,14 @@ The save function will look like this:
         };
     }
 
+    // ISaveable.cs
+    // Define an interface to be used by the overarching Save() method
+    // which the Node you wish to persist will need to implement.
+    public interface ISaveable
+    {
+        public Godot.Collections.Dictionary<string, Variant> Save();
+    }
```

Comment:
I don't know if this is clear enough, since we never explicitly show the Node implementing the interface. Also, comments in code can't be translated so that may make the C# code less accessible.